### PR TITLE
Adding SCOS example module page

### PR DESCRIPTION
**Pull request by floriankraemer**
## PR Description

Adding a SCOS example modules page.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/example-modules.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
```
Comment (gerner-spryker) [inline docs/scos/dev/example-modules.md:1]: Sean has requested a legal review on the example modules, please wait until legal has confirmed that our overview is accurate.

File: docs/scos/dev/example-modules.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+## The difference between core and example modules
+
+**Core modules** provide generic, reusable functionality that benefits a wide range of projects. They offer a foundation for customization and serve as versatile building blocks.
```
Comment (gerner-spryker) [inline docs/scos/dev/example-modules.md:4]: I think the terminology of this document is not aligned with the public terminology, eg: "core", "project", "customer", "SCOS", "core quality", "project quality"

File: docs/scos/dev/example-modules.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+**Example Modules** in contrast, are designed for specific, one-off use-cases. Their functionality is tightly aligned with unique business requirements and isn't suitable for broad reuse. These modules serve as specialized, project-specific solutions.
+
+## Example module use-cases
+
+### SCOS: Showcase functionality
+
+SCOS uses example modules to fill the gap with concrete/example implementation over an abstract workflow (latter is the target of delivery), where the concrete/example implementation is never actually usable by any concrete customer or business use-case, thus the actual business logic is not part of the core (eg: Example picking strategy).
```
Comment (gerner-spryker) [inline docs/scos/dev/example-modules.md:12]: Instead of the "example picking strategy", let's use another example that is better known

File: docs/scos/dev/example-modules.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+### SCOS: Unique business use-case
+
+SCOS uses example modules to implement functionality that is unique business (customer) use-case specific. The actual business logic is not part of the core as it does NOT offer regular core quality and is extracted into an example module.
+
+## Obligations
```
Comment (gerner-spryker) [inline docs/scos/dev/example-modules.md:18]: - This title is too legal.
- Some items are internal guidelines only.
- Some items are too direct.

**Review by gerner-spryker — CHANGES_REQUESTED** [review]

File: docs/scos/dev/example-modules.md (hunk 1 above)
Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:1]: I've just requested the legal review 1 minute ago. It will probably take legal some time to answer. My last request took ~2 weeks.

File: docs/scos/dev/example-modules.md (hunk 2 above)
Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:4]: OK, then let's use the React API example, because the Blog Example does not really fit the definition of "nonreuseable buisness logic" modules very well. All of the better example modules are new and mostly related to unified commerce.

My original idea was to add a list of modules, but this needs to be done after we fixed the readmes and license files for the old modules.

File: docs/scos/dev/example-modules.md (hunk 3 above)
Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:12]: OK, then let's use the React API example, because the Blog Example does not really fit the definition of "nonreuseable buisness logic" modules very well. All of the better example modules are new and mostly related to unified commerce.

My original idea was to add a list of modules, but this needs to be done after we fixed the readmes and license files for the old modules.

File: docs/scos/dev/example-modules.md (hunk 2 above)
Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:4]: What is the public terminology I have to use here? 😕

File: docs/scos/dev/example-modules.md (hunk 4 above)
Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:18]: True, yes, I should have caught that... The next commit will address this.

Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:18]: Updated

File: docs/scos/dev/example-modules.md (hunk 2 above)
Comment (floriankraemer) [reply docs/scos/dev/example-modules.md]: @gerner-spryker I've asked @lenadoc and she doesn't know either what exactly you want to change here.

File: docs/scos/dev/example-modules.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+
+## The difference between core and example modules
```
Comment (gerner-spryker) [inline docs/scos/dev/example-modules.md:2]: Please also ask legal's opinion on this document, as it is very legal content with promises to customers.

Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:2]: This is WIP now, legal answered but needs some time to finish this.

File: docs/scos/dev/example-modules.md (hunk 2 above)
Comment (gerner-spryker) [reply docs/scos/dev/example-modules.md]: @floriankraemer Are the above listed terms well-known and generally used by the audience of docs.spryker.com and Lena?

File: shell/run.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (gerner-spryker) [inline shell/run.sh:1]: is this file normal? :)

File: docs/scos/dev/example-modules.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+### SCOS: Unique business use-case
+
+SCOS uses example modules to implement functionality that is unique business (customer) use-case specific. The actual business logic is not part of the core as it does NOT offer regular core quality and is extracted into an example module.
+
+## Obligations
+
+### Exclusions and Non-Commitments
+
+Sprkyer has no strict obligations regarding the maintenance, compatibility, flexibility, and extendibility of the example modules.
```
Comment (gerner-spryker) [inline docs/scos/dev/example-modules.md:22]: I have no good definition on the title, but the text should be more like the following

```
We believe in providing transparency and setting clear expectations for our open-source packages.
We want you to know that our approach is a bit different on example modules:

* Stability
** We aim to keep our packages in good shape, but we don't promise a stable release.

* Compatibility
** While we work to keep things compatible, we can't guarantee forward or backward compatibility.

* Learnability
** We provide documentation when we can, but it's not a firm commitment. We encourage community learning and support.

...

In a nutshell, think of our example packages as a playground - there's potential for fun and innovations -, explore at your own pace!

```

File: shell/run.sh (hunk 6 above)
Comment (floriankraemer) [reply shell/run.sh:1]: I've informed @lenadoc, and asked if she is OK with this. 

> Wow, thank you very much for the shell script! 

It makes running the command easier. I don't like to type that command manually each time or copy-paste it. 😺

File: docs/scos/dev/example-modules.md (hunk 5 above)
Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:2]: The test was reviewed by legal and I've got this as answer:
> Thanks for the feedback. Sounds/all looks good, thanks.

File: docs/scos/dev/example-modules.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+**Core modules** provide generic, reusable functionality that benefits a wide range of customers. They offer a foundation for customization and serve as versatile building blocks.
+
+**Example Modules** in contrast, are designed for specific, one-off use-cases. Their functionality is tightly aligned with unique business requirements and isn't suitable for broad reuse. These modules serve as specialized, project-specific solutions.
+
+## Example module use-cases
+
+### SCOS: Showcase functionality
+
+SCOS uses example modules to fill the gap with concrete/example implementation over an abstract workflow (latter is the target of delivery), where the concrete/example implementation is never actually usable by any concrete customer or business use-case, thus the actual business logic is not part of the core (eg: the React API Example).
```
Comment (lenadoc) [inline docs/scos/dev/example-modules.md:12]: do you mean that the React API example is the example of an app with the example modules? Or anything else?

Comment (floriankraemer) [reply docs/scos/dev/example-modules.md:12]: Yes, but I just realized it was removed. I'll update this and use another module here.

**Comment by lenadoc** [comment]
@floriankraemer Please use this PR https://github.com/spryker/spryker-docs/pull/2434 for further updates of this document. As your PR is from your forked repository, and I cannot process it is a right way
